### PR TITLE
registry.k8s.io: bump archeio to v0.0.4

### PR DESCRIPTION
**Pull request by ameukam**

### CODE REVIEW

**Comment by ameukam** [comment]
/hold

For Monday rollout.

cc @BenTheElder @dims

File: infra/gcp/terraform/k8s-infra-oci-proxy-prod/terraform.tfvars [hunk 1]
```diff
@@ -16,7 +16,7 @@ limitations under the License.
 
 domain     = "registry.k8s.io"
 project_id = "k8s-infra-oci-proxy-prod"
-tag        = "v20221005-v0.0.2-28-g52c8633"
+tag        = "v0.0.4"
```
Comment (BenTheElder) [inline infra/gcp/terraform/k8s-infra-oci-proxy-prod/terraform.tfvars:19]: let's pin the digest this time.
small bit of defense in depth.

Comment (ameukam) [reply infra/gcp/terraform/k8s-infra-oci-proxy-prod/terraform.tfvars]: Fixed.

File: infra/gcp/terraform/k8s-infra-oci-proxy-prod/oci-proxy.tf [hunk 2]
```diff
@@ -99,7 +99,7 @@ resource "google_cloud_run_service" "oci-proxy" {
 
         // ensure this macth the value for template.spec.containers.resources.limits
         env {
-          name = "GOMAXPROCS"
+          name  = "GOMAXPROCS"
```
Comment (dims) [inline infra/gcp/terraform/k8s-infra-oci-proxy-prod/oci-proxy.tf:102]: unrelated space change?

Comment (ameukam) [reply infra/gcp/terraform/k8s-infra-oci-proxy-prod/oci-proxy.tf:102]: This is a result of running a lint command with the Terraform binary: `terraform fmt .`.

**Comment by ameukam** [comment]
Deployed:

```bash
gcloud run services list --project k8s-infra-oci-proxy-prod --format 'table(metadata.name,template.spec.containers.image)'
NAME                                           IMAGE
k8s-infra-oci-proxy-prod-asia-east1            ['us-central1-docker.pkg.dev/k8s-artifacts-prod/images/infra-tools/archeio@sha256:fc811af33c78e9765ee95d6994a6e0dac4c382ae63a960872404b7190110c769']
k8s-infra-oci-proxy-prod-asia-northeast1       ['us-central1-docker.pkg.dev/k8s-artifacts-prod/images/infra-tools/archeio@sha256:fc811af33c78e9765ee95d6994a6e0dac4c382ae63a960872404b7190110c769']
k8s-infra-oci-proxy-prod-asia-northeast2       ['us-central1-docker.pkg.dev/k8s-artifacts-prod/images/infra-tools/archeio@sha256:fc811af33c78e9765ee95d6994a6e0dac4c382ae63a960872404b7190110c769']
k8s-infra-oci-proxy-prod-asia-south1           ['us-central1-docker.pkg.dev/k8s-artifacts-prod/images/infra-tools/archeio@sha256:fc811af33c78e9765ee95d6994a6e0dac4c382ae63a960872404b7190110c769']
k8s-infra-oci-proxy-prod-australia-southeast1  ['us-central1-docker.pkg.dev/k8s-artifacts-prod/images/infra-tools/archeio@sha256:fc811af33c78e9765ee95d6994a6e0dac4c382ae63a960872404b7190110c769']
k8s-infra-oci-proxy-prod-europe-north1         ['us-central1-docker.pkg.dev/k8s-artifacts-prod/images/infra-tools/archeio@sha256:fc811af33c78e9765ee95d6994a6e0dac4c382ae63a960872404b7190110c769']
k8s-infra-oci-proxy-prod-europe-southwest1     ['us-central1-docker.pkg.dev/k8s-artifacts-prod/images/infra-tools/archeio@sha256:fc811af33c78e9765ee95d6994a6e0dac4c382ae63a960872404b7190110c769']
k8s-infra-oci-proxy-prod-europe-west1          ['us-central1-docker.pkg.dev/k8s-artifacts-prod/images/infra-tools/archeio@sha256:fc811af33c78e9765ee95d6994a6e0dac4c382ae63a960872404b7190110c769']
k8s-infra-oci-proxy-prod-europe-west2          ['us-central1-docker.pkg.dev/k8s-artifacts-prod/images/infra-tools/archeio@sha256:fc811af33c78e9765ee95d6994a6e0dac4c382ae63a960872404b7190110c769']
k8s-infra-oci-proxy-prod-europe-west4          ['us-central1-docker.pkg.dev/k8s-artifacts-prod/images/infra-tools/archeio@sha256:fc811af33c78e9765ee95d6994a6e0dac4c382ae63a960872404b7190110c769']
k8s-infra-oci-proxy-prod-europe-west8          ['us-central1-docker.pkg.dev/k8s-artifacts-prod/images/infra-tools/archeio@sha256:fc811af33c78e9765ee95d6994a6e0dac4c382ae63a960872404b7190110c769']
k8s-infra-oci-proxy-prod-europe-west9          ['us-central1-docker.pkg.dev/k8s-artifacts-prod/images/infra-tools/archeio@sha256:fc811af33c78e9765ee95d6994a6e0dac4c382ae63a960872404b7190110c769']
k8s-infra-oci-proxy-prod-southamerica-west1    ['us-central1-docker.pkg.dev/k8s-artifacts-prod/images/infra-tools/archeio@sha256:fc811af33c78e9765ee95d6994a6e0dac4c382ae63a960872404b7190110c769']
k8s-infra-oci-proxy-prod-us-central1           ['us-central1-docker.pkg.dev/k8s-artifacts-prod/images/infra-tools/archeio@sha256:fc811af33c78e9765ee95d6994a6e0dac4c382ae63a960872404b7190110c769']
k8s-infra-oci-proxy-prod-us-east1              ['us-central1-docker.pkg.dev/k8s-artifacts-prod/images/infra-tools/archeio@sha256:fc811af33c78e9765ee95d6994a6e0dac4c382ae63a960872404b7190110c769']
k8s-infra-oci-proxy-prod-us-east4              ['us-central1-docker.pkg.dev/k8s-artifacts-prod/images/infra-tools/archeio@sha256:fc811af33c78e9765ee95d6994a6e0dac4c382ae63a960872404b7190110c769']
k8s-infra-oci-proxy-prod-us-east5              ['us-central1-docker.pkg.dev/k8s-artifacts-prod/images/infra-tools/archeio@sha256:fc811af33c78e9765ee95d6994a6e0dac4c382ae63a960872404b7190110c769']
k8s-infra-oci-proxy-prod-us-south1             ['us-central1-docker.pkg.dev/k8s-artifacts-prod/images/infra-tools/archeio@sha256:fc811af33c78e9765ee95d6994a6e0dac4c382ae63a960872404b7190110c769']
k8s-infra-oci-proxy-prod-us-west1              ['us-central1-docker.pkg.dev/k8s-artifacts-prod/images/infra-tools/archeio@sha256:fc811af33c78e9765ee95d6994a6e0dac4c382ae63a960872404b7190110c769']
k8s-infra-oci-proxy-prod-us-west2              ['us-central1-docker.pkg.dev/k8s-artifacts-prod/images/infra-tools/archeio@sha256:fc811af33c78e9765ee95d6994a6e0dac4c382ae63a960872404b7190110c769']
```
